webpack.config.client.js

```diff
@@ -178,14 +192,18 @@ module.exports = ({
         // Display full duplicates information? (Default: `false`)
         verbose: true,
       }),
+      /*
+       * webpack 5 does no longer includes a polyfill for the Node.js process variable in
+       * frontend code. webpack advise to avoid using it in the frontend code however the
+       * following plugin will enable the process variable in frontend code until we find
+       * an alternative for this sort of thing.
+       */
+      new webpack.ProvidePlugin({
+        process: 'process/browser',
+      }),
       new webpack.DefinePlugin({
         'process.env': getClientEnvVars(DOT_ENV_CONFIG),
       }),
-      /**
-       * Needed to prevent bundle hashes changing when the order they're imported is changed.
-       * See https://webpack.js.org/guides/caching/#module-identifiers
-       */
-      new webpack.HashedModuleIdsPlugin(),
```

Comment:
From [the update-outdated-options section](https://webpack.js.org/migrate/5/#update-outdated-options) of the webpack upgrade docs 

> HashedModuleIdsPlugin → optimization.moduleIds: 'deterministic'